lms/db/__init__.py

```diff
@@ -102,6 +104,13 @@ def init(engine, drop=False):
     BASE.metadata.create_all(engine)
 
 
+def stamp_db(engine):  # pragma: nocover
```

Comment:
I don't think it's worth adding unit tests for something that's only used in dev / I think the unit test you'd likely write for this would probably be a bit silly

Reply:
I'm pretty sure we'll test it practically enough by running it a million times a day.

Add a docstring saying what this does and that it's only to be used in dev? Also is this private? Should it be '_stamp_db'?

Reply:
Added the `_` and a docstring